### ISSUE TITLE
Qt6: Table view header: invisble sort indicator occupies space = overlaps titles

### ISSUE THREAD

**Issue by ronso0** [mixxxdj/mixxx#13373]
### Bug Description

Qt 5.12.8
![Screenshot_2024-06-14_23-49-28](https://github.com/mixxxdj/mixxx/assets/5934199/5118c11f-6f13-43d2-9a98-8f9f111093c3)

Qt 6.2.3
![Screenshot_2024-06-14_23-50-58](https://github.com/mixxxdj/mixxx/assets/5934199/2c2e1ea6-585a-4b51-9ee0-3527e35a2c43)

Does anyone also see this with a Qt version newer than 6.2.3?

I tried setting the indicator width via qss, didn't work (don't even know if the selectors would apply to QHeaderView)
```
WTrackTableViewHeader::up-arrow:disabled,
WTrackTableViewHeader::up-arrow:off,
WTrackTableViewHeader::up-arrow:!enabled {
  width: 0px;
}
```

### Version

2.5 and newer

**Comment by JoergAtGithub** [comment]
This happens also with Qt6.5 under Windows

**Comment by ywwg** [comment]
Is this still happening? @JoergAtGithub @ronso0

**Comment by JoergAtGithub** [comment]
I just tried this with a recent main build using Qt6.5 under Windows11, and it no longer happens.

**Comment by ronso0** [comment]
I fixed it with #13535
